arbitrum-docs/run-arbitrum-node/sequencer/04-run-sequencer-node.mdx

```diff
@@ -0,0 +1,199 @@
+--node.batch-poster.parent-chain-wallet.private-key=<Your Parent Chain Wallet Private Key>
+```
+
+#### 4. Disable transaction forwarding
+
+Disable your sequencer's forwarding transaction as the node will queue the transaction directly:
+
+```shell
+--execution.forwarding-target=""
+```
+
+#### 5. Enable feed out queued transaction
+
+Enable your node to feed out transaction so fullnode can receive queued transaction:
+
+```shell
+--node.feed.output.enable=true
+--node.feed.output.addr=<Sequencer feed url>
+--node.feed.output.port=<Sequencer feed port>
+```
+
+#### 5. Connect the node to data availability servers
+
+(This step only required in Anytrust mode)
+Enable your node can send batch to DAS and get DACerts from them.
+
+```shell
+--node.data-availability.enable=true
+--node.data-availability.sequencer-inbox-address=<Sequencer Inbox Address>
+--node.data-availability.parent-chain-node-url=<Parent Chain Node URL>
+--node.data-availability.rest-aggregator.enable=true
+--node.data-availability.rest-aggregator.urls=<A list of DAS REST endpoints, can be only one URL>
+--node.data-availability.rpc-aggregator.enable=true
+--node.data-availability.rpc-aggregator.assumed-honest=1
+--node.data-availability.rpc-aggregator.backends=<A list of RPC backends>
+```
+
+### 2. Putting it all together
+
+- When running a Docker image, an external volume should be mounted to persist the database across restarts. The mount point inside the Docker image should be `/home/user/.arbitrum`
+
+- Example:
+
+  ```shell wordWrap=true
+  docker run --rm -it -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 @@latestNitroNodeImage=offchainlabs/nitro-node:v3.5.1-8f247fd@@ --node.sequencer=true --node.delayed-sequencer.enable=true --node.delayed-sequencer.use-merge-finality=false --node.delayed-sequencer.finalize-distance=1 --node.batch-poster.enable=true --node.batch-poster.max-size=90000 --node.batch-poster.parent-chain-wallet.private-key=<Your Parent Chain Wallet Private Key> --node.staker.enable=true --node.staker.strategy=MakeNodes --node.staker.parent-chain-wallet.private-key=<Your Parent Chain Wallet Private Key> --node.data-availability.enable=true --node.data-availability.sequencer-inbox-address=<Sequencer Inbox Address> --node.data-availability.parent-chain-node-url=<Parent Chain Node URL> --node.data-availability.rest-aggregator.enable=true --node.data-availability.rest-aggregator.urls=<A list of DAS REST endpoints, can be only one URL> --node.data-availability.rpc-aggregator.enable=true --node.data-availability.rpc-aggregator.assumed-honest=1 --node.data-availability.rpc-aggregator.backends=<A list of RPC backends> --execution.sequencer.enable=true --execution.sequencer.max-tx-data-size=85000 --execution.sequencer.max-block-speed=250ms
+  ```
+
+  - Ensure that `/some/local/dir/arbitrum` already exists otherwise the directory might be created with `root` as owner, and the Docker container won't be able to write to it
```

Comment:
```suggestion
  - Ensure that `/some/local/dir/arbitrum` already exists otherwise, the directory might be created with `root` as owner, and the Docker container won't be able to write to it.
```